tests/karate/src/test/java/be/constants.feature

```diff
@@ -0,0 +1,36 @@
+@ignore @report=false
+Feature: Constants
+  Scenario: Creates constants that will be used by other features
+    # TODO: make the function depend on all the attributes the lao id depends on
```

Comment:
I'm thinking it might make sense to have a generic `HashLen` function in Java, and then maybe a `fillValidLaoId` / `fillValidElectionId` / etc. functions in JS